### PR TITLE
New template surf_mvp_coordinatable_alert

### PR DESCRIPTION
**Pull request by serge-nanaev**
New template for creating Alert modules in project using coordinated Surf MVP architecture

### CODE REVIEW

**Comment by ismetanin** [comment]
Does this template differ with `surf_mvp_coordinatable_module` by only one line?

File: surf_mvp_coordinatable_alert/Code/Configurator/module_configurator.swift.liquid [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import UIKit
+
+final class {{ module_info.name }}AlertConfigurator {
+
+    // MARK: - Internal methods
+
+    func configure() -> ({{ module_info.name }}Controller, {{ module_info.name }}ModuleOutput) {
+
+        // TODO: Change title and message
```
Comment (ismetanin) [inline surf_mvp_coordinatable_alert/Code/Configurator/module_configurator.swift.liquid:17]: Maybe do this now?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ The detailed information about a template structure is available in [Generamba W
 
 * [Surf MVP Coordinatable module](https://github.com/surfstudio/generamba-templates/tree/master/surf_mvp_coordinatable_module) - generates a new module of architecture **Surf MVP** with routing using Coordinators.
 
+* [Surf MVP Coordinatable Alert module](https://github.com/surfstudio/generamba-templates/tree/master/surf_mvp_coordinatable_alert) - generates a new alert module of architecture **Surf MVP** with routing using Coordinators.
```
Comment (ismetanin) [inline README.md:17]: I think it doesn't depend on coordinators, so we can name this template like "Surf MVP Alert module"

**Review by ismetanin — CHANGES_REQUESTED** [review]

File: surf_mvp_coordinatable_alert/Code/Configurator/module_configurator.swift.liquid (hunk 1 above)
Comment (serge-nanaev) [reply surf_mvp_coordinatable_alert/Code/Configurator/module_configurator.swift.liquid:17]: This TODO must be resolved in project code for every alert module.

File: README.md (hunk 2 above)
Comment (serge-nanaev) [reply README.md:17]: I think it depends on coordinators.
1. It have no Router
2. Presenter is realization of ModuleOutput, what is incorrect in normal MVP module.

Comment (ismetanin) [reply README.md:17]: @s4x Oh, I see, okay

**Review by ismetanin — APPROVED** [review]

**Pull request merged**